src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <!-- Mobile and legacy targets - comment these out or set env variable MSAL_DESKTOP_ONLY_DEV to speedup the build -->
-  <PropertyGroup Condition="'$(MSAL_DESKTOP_ONLY_DEV)' == ''">
+  <PropertyGroup Condition="'$(MSAL_DESKTOP_ONLY_DEV)' != ''">
```

Comment:
Consider renaming this to `INCLUDE_MOBILE_AND_LEGACY_TFM` or smth like that. I think the builds need to change as well, as they'll have to set this property.

Reply:
Yes, variables can be added to the UI on ADO build config page.